independent-projects/tools/analytics-common/src/main/java/io/quarkus/analytics/AnalyticsService.java

```diff
@@ -258,8 +258,8 @@ private boolean allEnvSet(String... names) {
     }
 
     private String getQuarkusVersion(ApplicationModel applicationModel) {
-        return applicationModel.getPlatforms().getImportedPlatformBoms().stream()
-                .filter(artifactCoords -> artifactCoords.getArtifactId().equals("quarkus-bom"))
+        return applicationModel.getDependencies().stream()
+                .filter(artifactCoords -> artifactCoords.getArtifactId().equals("quarkus-bootstrap-core"))
```

Comment:
What do you think @aloubyansky ?

Reply:
I wouldn't do this, assuming we want the platform version. But I'm curious why the version is not detectable. It also depends what we want as "Quarkus version".

The difference is that, currently it's the platform version and while this will be the core, that may not be aligned downstream, just in case.

Second point, it's not exactly about direct `quarkus-bom` imports. Even there is a hierarchy of parent POMs and BOM imports, including flattening, the `quarkus-bom` should be detectable unless some (fingerprinting) part of it is removed.

Reply:
I agree there are too many.